public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -79,12 +79,7 @@ export class TimeSrv {
       return intervals;
     }
 
-    const validIntervals = intervals.filter(str => str !== '').filter(this.contextSrv.isAllowedInterval);
-
-    if (validIntervals.indexOf(this.contextSrv.minRefreshInterval) === -1) {
-      validIntervals.unshift(this.contextSrv.minRefreshInterval);
```

Comment:
not sure why this logic was added here, @marefr do you know?

Reply:
No, not sure. If I understand it correctly this adds the min interval to valid intervals if it doesn't exists? Yeah that sounds wrong to me.